Example/GenericValidatorExampleTests/StringEvaluationTests.swift

```diff
@@ -91,9 +91,9 @@ class StringEvaluationTests: XCTestCase {
         XCTAssertTrue(text.isEmailValid())
     }
 
-    func test_whenTextIsValidEmail_longExtension() {
+    func test_whenTextIsInvalidEmail_multipleDomainExtension() {
         let text = "marc@aol.co.uk"
```

Comment:
@htinlinn good point, I pushed an update of the email regular expression to accept multiple domains